src/pages/PetitionPage/PetitionPage.tsx

```diff
@@ -13,39 +13,34 @@ const cx = classNames.bind(styles);
 
 function PetitionPage() {
   const [downState, setDownState] = useState(false);
-  const callback = entries => {
-    entries.forEach(entry => {
-      if (entry.isIntersecting) {
-        setDownState(false);
-      } else {
-        setDownState(true);
-      }
-    });
-  };
-  const observer = new IntersectionObserver(callback, {
-    rootMargin: '-100px -100px',
-  });
+  const titleRef = useRef<any>(null);
```

Comment:
```suggestion
  const titleRef = useRef<HTMLHeadingElement>(null);
```
로 선언해주면 될것같아요 HTML{태그명}Element (ex. HTMLDivElement, HTMLImageElement, ...)